test/gtest/uct/test_p2p_rma.cc

```diff
@@ -62,7 +62,7 @@ ucs_status_t uct_p2p_rma_test::get_zcopy(uct_ep_h ep, const mapped_buffer &sendb
 void uct_p2p_rma_test::test_xfer(send_func_t send, size_t length,
                                  unsigned flags, uct_memory_type_t mem_type)
 {
-    mapped_buffer sendbuf(length, SEED1, sender(), 1);
+    mapped_buffer sendbuf(length, SEED1, sender(), 1, mem_type);
```

Comment:
I think this might break existing cuda uct gdrcopy, cuda_copy as there are expecting to send buffer is host buffer.  Let me think how to handle this. I think we need a way cuda TL src buffer can HOST or CUDA